services/explorer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@taraxa_project/explorer",
+  "description": "Taraxa Explorer",
+  "version": "0.0.1",
+  "main": "template.json",
+  "license": "MIT",
+  "dependencies": {
+    "@material-ui/core": "^4.12.2",
```

Comment:
Why do we have @material-ui version 4 here and also @mui/material? We should remove the old version but make sure the new version has all it's dependencies like 'lab' and 'icons'